ops/pebble.py

```diff
@@ -2979,6 +2979,9 @@ def exec(
         below, however, input/output handling is a bit more complex. Some
         examples are shown below::
 
+            >>> import pytest
+            >>> pytest.skip('these examples require a running Pebble')
```

Comment:
What do you think about changing the format of examples that we don't want to be doctests instead? For example, assuming it still renders nicely in the docs and actually disables doctests, a single leading `>` instead of `>>>`? That said, your approach here is nice and explicit, not too intrusive, and minimises diffs, so I'm happy to be overruled here.

Reply:
I wasn't familiar with doctests until today, so I do think that

```
import pytest
pytest.skip('these examples require a running Pebble')
```

at the top of the example would have been a significant point of confusion.

I'm OK if this is the most practical solution. How about including a comment to make it even more explicit:

```suggestion
            # These two lines are related to documentation testing, not exec()
            >>> import pytest
            >>> pytest.skip('these examples require a running Pebble')
```

Is there a convenient way to disable doctests for pebble.py only? I'd prefer that if possible. (Sorry if we discussed that possibility during standup and I missed it)

Or inspired by James's suggestion, if we keep doctests for pebble.py, could we drop `>>>` entirely and use comments to explain the output of each example? This might work well, as we only need to show a small amount of output for each example.

Reply:
> Is there a convenient way to disable doctests for pebble.py only?

There's not for this specific docstring. For all of pebble.py, it depends on how you define "convenient" :smile:. For example, if we ran the doctests separately then we could only include the specific files, but I think it's better all in the single pytest command. I'm also not keen on excluding the whole module when we might want to use doctest on other docstrings in here in the future.

> could we drop >>> entirely and use comments to explain the output of each example? This might work well, as we only need to show a small amount of output for each example.

That looks like this for the simpler cases:

<img width="960" height="824" alt="image" src="https://github.com/user-attachments/assets/86035939-9d0a-4ff8-ad34-933a6d05ccb6" />

(I do prefer just dropping the `>>>` over using `>`).

It's not as nice for the more complicated ones, like multi-line output and exceptions:

<img width="952" height="1016" alt="image" src="https://github.com/user-attachments/assets/4c687e80-3ee0-4662-b903-4029b9897dd0" />

And for completeness, with the comment it's like this:

<img width="952" height="280" alt="image" src="https://github.com/user-attachments/assets/04433017-f772-4793-893a-a537ceabd025" />

Thoughts?